content/_img-sources/amm-concept-deposit.uxf

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
These three diagram files are currently unused. The hope is for a follow-up to add diagrams based on/inspired by these. Here's what the rough draft of this one looks like:
![amm-concept-deposit](https://user-images.githubusercontent.com/7515597/207204153-8ec77ae7-3f73-4f22-b056-3972891835f4.png)